weave/weave_server.py

```diff
@@ -347,6 +348,11 @@ def hello():
     return "hello"
 
 
+@blueprint.route("/__weave/version")
+def version():
+    return __version__
```

Comment:
This is fine for now but may still cause the original confusion that led to this PR  because we dont just deploy releases, we also deploy commit hashes in between releases. So it would be better if we could show that, but this is an improvement over what was there before (just showing 0.0.6 🙃 ).